client/client.go

```diff
@@ -681,27 +678,31 @@ func (c *client) handleDispatcher(
 	if dc == globalDCLocation {
 		go func() {
 			var updateTicker = &time.Ticker{}
-			if c.option.getTSOFollowerProxyOption() {
-				updateTicker = time.NewTicker(memberUpdateInterval)
-				defer updateTicker.Stop()
+			setNewUpdateTicker := func(ticker *time.Ticker) {
+				if updateTicker.C != nil {
+					updateTicker.Stop()
+				}
+				updateTicker = ticker
 			}
+			defer setNewUpdateTicker(nil)
```

Comment:
Here is going to return, what is the meaning of setting this local variable `updateTicker`?

Reply:
add comment for it